static/app/views/dashboards/widgetCard/spansWidgetQueries.tsx

```diff
@@ -283,6 +288,13 @@ function SpansWidgetQueriesSingleRequestImpl({
         dashboardFilters={dashboardFilters}
         onDataFetched={onDataFetched}
         afterFetchSeriesData={afterFetchSeriesData}
+        samplingMode={
+          organization.features.includes(
+            'visibility-explore-progressive-loading-normal-sampling-mode'
+          )
+            ? SAMPLING_MODE.NORMAL
+            : undefined
+        }
```

Comment:
If you find yourself doing this a lot, i might suggest creating another hook, `useExploreSamplingMode` which does the feature check and returns the appropriate sampling mode